doc/Learn/Maui/MauiOverview.md

```diff
@@ -49,6 +51,28 @@ dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 
 .NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
 
+### MauiEmbedding Dependencies
+
+# [**Single Project**](#tab/single-project)
+
+![Starting solution structure - SingleProject](Assets/GettingStartedExisting01-StartSingleProjectStructure.png)
+
+**UnoFeatures**
+In the class library find the `<UnoFeatures>` property and add `MauiEmbedding`. This step add all the dependencies you need for you to have MauiEmbedding working with you project.
+
+```diff
+<UnoFeatures>
+  ...
+  Serialization;
+  Localization;
+  Navigation;
+  ExtensionsCore;
++ MauiEmbedding;
+</UnoFeatures>
+```
+
+# [**Multi-Head (Legacy)**](#tab/multi-project)
```

Comment:
```suggestion
#### [**Multi-Head Project Template (Legacy)**](#tab/multi-project)
```

Reply:
@jeromelaban, @kazo0 let's agree on something consistent that we can re-use across the doc :)
![image](https://github.com/unoplatform/uno.extensions/assets/16295702/fcf6cb11-7f20-48da-b098-ab1a9d488657)
(cf. https://github.com/unoplatform/Uno.Themes/pull/1385/files)